tests/conftest.py

```diff
@@ -63,6 +64,28 @@ def sqlite_url(tmpdir):
     return "sqlite:///%s" % path
 
 
+@pytest.fixture
+def sqlite_max_variable_number():
+    # Return SQLite max. variable number limit, set as compile option, or
+    # default.
+    #
+    # Default value for stock SQLite >= 3.32.0
+    # (https://www.sqlite.org/limits.html#max_variable_number): 32766
+    # 
+    # Note that distributions *do* customize this e.g. Ubuntu 20.04:
+    # MAX_VARIABLE_NUMBER=250000
+    conn = sqlite3.connect(':memory:')
```

Comment:
May be use the common function `create_engine` to create connection object:

```
engine = sa.create_engine(sqlite_url)
conn = engine.connect()
```

Reply:
I actually tried s.th. along those lines but ran into difficulties. SQLAlchemy seems to use some intricate connection/session state handling
(I admit I do not really understand that yet) which somehow caused the execution of the 'pragma compile options;' to get performed twice, which made it unpredictable to me how to get at the desired MAX_VARIABLE_NUMBER value.

Probably there would have been some easy remedy for it - I just could not quickly find it and decided I couldn't be bothered: Since this is a throwaway execution of an SQLite-specific command to get at a compile option, usage of a full blown SQLAlchemy session, db-agnostic and bells and whistles, does not really provide any benefits.

By going directly through sqlite3 I completely avoid interfering with any SQLAlchemy connection/session state.

So I'm hesitant to change that to using the SQLAlchemy API here. I could at a comment to explain the motivation of
using `sqlite3.connect(':memory:')`?

Reply:
Thanks! that sounds reasonable! We can keep this change for later.